### ISSUE TITLE
[ERROR] While using debug bar other package

### ISSUE THREAD

**Issue by zulfikar-ditya** [protonemedia/laravel-splade#409]
- Laravel Version: 10.9.0
- PHP Version: 8.1.12
- Splade JS Version (npm): 1.4.7
- Splade PHP Version (composer): 1.4.7
- Dev environment (OS, Sail/Valet/etc): Window 11 Pro 64 bit

I use [Laravel debug bar](https://github.com/barryvdh/laravel-debugbar) for my development process, it get error dynamicProps like in #185 . Like the protonemedia said we can't use `script` tag in our blade, but must use `<x-splade-script>...</x-splade-script>` like in #181

Any options or help to this issue ?

Thanks

**Comment by pascalbaljet** [comment]
Are you sure this is caused by the Debug Bar package? I'm using it as well and don't have any problems with it. Can you try to disable it and see if the error still occurs?

**Comment by zulfikar-ditya** [comment]
Yes. When I disable using .env file or from config Splade work fine with no errors
